qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
Why? This is "Default Language Contributors", not translations, you are the one who introduced 'pt-br'? As the translator of pt_PT, you are mentioned below. At some point I will probably move it to web page, it does not really belong to .php file, do you think so?